dbfit-java/db2/src/main/java/dbfit/environment/DB2Environment.java

```diff
@@ -79,11 +79,11 @@ protected String getConnectionString(String dataSource, String database) {
                 // int length=rs.getInt(3);
                 String direction = rs.getString(4);
                 Direction paramDirection = getParameterDirection(direction);
-                DbParameterAccessor dbp = new DbParameterAccessor(paramName,
-                        paramDirection, getSqlType(dataType),
-                        getJavaClass(dataType),
-                        paramDirection == RETURN_VALUE ? -1
-                                : position++);
+                DbParameterAccessor dbp = createDbParameterAccessor(paramName,
+                                                                    paramDirection,
```

Comment:
With this alignment one of the lines gets harder to read (I need to scroll :-)). If we really want to align all - we may rather move paramName down and keep the rest as before.